src/local_identifiability.jl

```diff
@@ -135,13 +135,18 @@ function get_degree_and_coeffsize(f::Generic.Frac{<: MPolyElem{Nemo.fmpq}})
 end
 
 # ------------------------------------------------------------------------------
-function assess_local_identifiability(ode::ModelingToolkit.ODESystem, inputs=[], funcs_to_check=[], p::Float64=0.99, type=:SE)
-    ode, syms, gens_ = PreprocessODE(ode, inputs)
+function assess_local_identifiability(ode::ModelingToolkit.ODESystem, p::Float64=0.99, type=:SE)
+    ode, syms, gens_ = PreprocessODE(ode)
+    return assess_local_identifiability(ode, ode.parameters, p, type)
+end
+
+function assess_local_identifiability(ode::ModelingToolkit.ODESystem, funcs_to_check=[], p::Float64=0.99, type=:SE)
+    ode, syms, gens_ = PreprocessODE(ode)
     if length(funcs_to_check) > 0
         funcs_to_check = [eval_at_nemo(x, Dict(syms .=> gens_)) for x in funcs_to_check]
-        return assess_local_identifiability(ode, funcs_to_check, p, type)
+        return assess_local_identifiability(ode, funcs_to_check, p, type) 
     else
-        return assess_local_identifiability(ode, p, type)
+        return assess_local_identifiability(ode, ode.parameters, p, type)
```

Comment:
I believe this will return in the `Nemo` format, not in the MTK, won't it?

Reply:
Yes, but local identifiability checks don't return dictionaries, it will be an array of `Bool`

Reply:
oh, right!